### DISCUSSION TITLE
Can't see where emails are being sent when using route with multiple recipients

### DISCUSSION THREAD

**Discussion by Fornax96** [postalserver/postal#1489] [Feature suggestions]
Hi. First of all thanks for developing postal. Been using it for a few days now and it seems like a great piece of software.

I have a route with a bunch of e-mail address endpoints in it, and when I send a mail to that route it gets forwarded to all those endpoints as expected. But one of the target servers is rejecting my messages and I get a bounce message back every time. In the postal UI I can't find anywhere which address is causing the error because all the messages show the same From and To addresses.

Example:
![image](https://user-images.githubusercontent.com/1247928/82755253-bc88db00-9dd2-11ea-875f-6a3241b13ad7.png)

As you can see one of the messages failed, but I can't see which address is at fault.

Another strange thing is that all the messages show as incoming messages when they are in fact outgoing messages.

**Comment by willpower232** [comment]
If you go into the message in the UI, you should be able to get more detail about that particular message including the error from the receiving server and the email headers that were present.

Incoming means that the messages were received courtesy of some MX records rather than using the SMTP credential so it might be that your setup isn't quite right somehow.

**Comment by Fornax96** [comment]
The mail was forwarded through a route, so that makes sense.

But in the message details it only shows that the message was sent to the route address. In the technical details of the bounce I can see that it was sent to a gmail server, but not which address. Even in the email headers it just says `To: <route address>`, but not the address it was being forwarded to.

**Comment by willpower232** [comment]
I see. The technical information in the UI for the message that you have to expand will include a support ref which should correspond to some log entries so I'd hope you can see the minute detail there?

**Comment by Fornax96** [comment]
I was able to find the failing address in the SMTP log. But I really think this should be added to the UI somewhere.

And I know that postal was made for sending, and not routing. But even though it was not the intention, postal is the best solution for this use case right now. I couldn't find any other alternative for my routes which was as easy to configure as postal.

**Comment by willpower232** [comment]
I agree, will mark as feature request
